ports/atmel-samd/boards/arduino_mkrzero/mpconfigboard.mk

```diff
@@ -11,3 +11,6 @@ LONGINT_IMPL = NONE
 CIRCUITPY_FULL_BUILD = 0
 
 SUPEROPT_GC = 0
+
+MICROPY_PY_ASYNC_AWAIT = 0
+
```

Comment:
To placate the so-called "pre-commit hook", these added trailing newlines must be deleted.

Would the total number of changes be smaller if you made CIRCUITPY_FULL_BUILD = 0 imply MICROPY_PY_ASYNC_AWAIT = 0?